test-network-function/generic/suite.go

```diff
@@ -935,10 +935,18 @@ func podAntiAffinity(deployment, podNamespace string, replica int) {
 
 	testResult, err := tester.Run()
 	if testResult != tnf.SUCCESS {
-		msg := fmt.Sprintf("The POD replica is found %d, and podAntiAffinity rule is not defined, you might want to change it in deployment %s in namespace %s",
-			replica, deployment, podNamespace)
-		log.Warn(msg)
-		infoWriter(msg)
+		if replica > 1 {
```

Comment:
Good change. I was thinking about this too. Again, if we had two separate check functions, this wouldn't feel like additional work.